jbmc/src/java_bytecode/java_utils.cpp

```diff
@@ -191,6 +191,12 @@ dereference_exprt checked_dereference(const exprt &expr)
   return result;
 }
 
+dereference_exprt
+element_at_pointer_array_index(const exprt &pointer, const exprt &index)
```

Comment:
Suggested name `array_element_from_pointer`

Reply:
Changed the name to your suggestion.